liquibase-core/src/main/java/liquibase/change/core/RenameSequenceChange.java

```diff
@@ -41,7 +42,7 @@ public void setSchemaName(String schemaName) {
         this.schemaName = schemaName;
     }
 
-    @DatabaseChangeProperty(mustEqualExisting = "sequence", description = "Name of the sequence to rename")
+    @DatabaseChangeProperty(mustEqualExisting = "sequence", description = "Name of the existing sequence to rename")
```

Comment:
:) We may have a different philosophical opinion on if "existing" is necessary or not.

Reply:
I think this change was partially just to maintain consistency ("existing" is used in a few places throughout the docs), though I have been trying to think about ways the descriptions can be misinterpreted by users whose first language isn't English. In this case, I was trying to contrast the `oldSequenceName` and `newSequenceName` descriptions. "Old" has a different meaning than "current"/existing, which is something native/proficient speakers would immediately understand, but it's possible that novice English speakers may not immediately make the connection. i.e. conceivably they might assume we're asking for a third (old) sequence name rather than a current one.

Hard to say.